raiden/tests/integration/long_running/test_settlement.py

```diff
@@ -382,14 +381,14 @@ def run_test_batch_unlock(
         in token_network.partneraddresses_to_channelidentifiers[alice_app.raiden.address]
     )
 
-    # wait for the node to call batch unlock
+    # Wait for both nodes to call batch unlock
```

Comment:
Below this, we're only waiting on bob's app to receive the batch unlock but not alice... we should do it twice there.

Reply:
Since the asserts after this wait depend on one side of the channel-state, we agreed not to do this.